api/envoy/extensions/transport_sockets/http_11_proxy/v3/upstream_http_11_connect.proto

```diff
@@ -17,11 +17,11 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#extension: envoy.transport_sockets.http_11_proxy]
 
 // Configuration for HTTP/1.1 proxy transport sockets.
-// If this is configured and an intermediate filter adds proxy metadata to the
-// stream info then
+// If this is configured and an intermediate filter adds the data necessary for
+// proxying to the stream info then
```

Comment:
yeah this is why I didn't want to do genreal Envoy docs for it for it as it's written for a very narrow use case.  I suspect it will get extended to be general purpose over time but right now, the goal is to provide the infrastructure for E-M sending data through the local android proxy (https://github.com/envoyproxy/envoy-mobile/issues/1622)  The E-M filter will query Android for the address and port of the proxy, then add that information to the stream info filter state so it can be used to redirect the connection and alter the payload going to the proxy.
I'm super happy to put these details in the proto but I've been repeatedly chided by @markdroth for putting too much Envoy information in what is supposed to be generic protos.  Given this one is Envoy-only maybe it's OK to add more detail there?  But this really isn't intended for generic Envoy use. I could also land the proto and extension in E-M but then I can't test any of the changes to core Envoy required to make it all hang together.  Happy to take your advice here.,

Reply:
OK I see. I don't know maybe then just make it a bit more clear in the docs that this is EM specific right now and can be extended later? 

I get the concern about Envoy specific info in the protos, but IMO 1st priority is documentation. We need to sort out the per client docs separately which is a known issue.